driver/registry_default_test.go

```diff
@@ -68,10 +68,10 @@ func TestDriverDefault_Strategies(t *testing.T) {
 	}{
 		{prep: func(conf *config.Config) {
 			conf.MustSet(config.ViperKeySelfServiceStrategyConfig+".password.enabled", false)
-		}},
+		}, expect: []string{"password", "oidc"}},
 		{prep: func(conf *config.Config) {
 			conf.MustSet(config.ViperKeySelfServiceStrategyConfig+".password.enabled", true)
-		}, expect: []string{"password"}},
+		}, expect: []string{"password", "oidc"}},
```

Comment:
The changes in this file should be reverted as this is still the expected behavior!